docs/scraping/batch-scraping.md

```diff
@@ -0,0 +1,36 @@
+# (Experimental Feature) Batch Scraping Through Dedicated Azure Monitor API
+
+Under batch scraping mode, Promitor will fetch resource metrics in batches, through dedicated Azure Monitor APIs. 
+
+Promitor deployments are able to monitor a greater set of unique resources and metrics, in part because its requests 
+go through dedicated APIs that allow for much greater throughput. However, there are important factors to consider(like cost) when
+deploying Promitor this way. 
+
+![Promitor Scraper with resource discovery](./../media/concepts/scraper-batch-scraping-mode.png)
+
+## Why Do Batch Scraping?
```

Comment:
I couldn't find anything in the public, unfortunately :/ 

But since the API is data plane not control plane(ARM), I'm fairly sure rate limit will be much more generous